sdk/storage/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "azure-storage" %}
+{% set version = "2021.05.01" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: {{ environ.get('STORAGE_SOURCE_DISTRIBUTION', '') }}
+
+build:
+  script_env:
+    - STORAGE_SOURCE_DISTRIBUTION
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - azure-core >=2021.05.01
```

Comment:
Do these versions have any connection to the pip package versions are will they always be independent? 

I assume this 2021.05.01 versioning scheme is just for your testing correct? If we end up having another versioning scheme for our conda packages we should get the information added to our versioning guidelines https://azure.github.io/azure-sdk/policies_releases.html#python.

Reply:
This is just for testing. Xiang and I believe that for the conda packages that are **directly equivalent** to their `pip` counterparts, we will version the same.

Reply:
This is an update that will come with the next PR on all this stuff. Need to figure out the publishing story. Will be updating these when that happens.